deployments/k8s/helm/signalfx-agent/restricted-sfx-agent.yaml

```diff
@@ -0,0 +1,81 @@
+# This Pod Security Policy is setup to allow the signalfx-agent daemonset to
```

Comment:
Looks like all of these need to be moved to templates and templatized.

Reply:
Closed #321 PR and merge to here. Let's finalize and wrap up task if possible. Let me also revise Jay's above comments. Following is PR #321 summary for reference. 

keitwb 29 days ago
Here is an example of conditionalizing the template. This is yaml, so if statements aren't allowed here.

hiragkumarp commented 11 days ago
Thanks Ben for giving hint. Sorry for late update. I also come across this Helm Flow control standard (https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md) and updated value YAML code accordingly with IF condition. Is this sounds good to approve?

keitwb commented 2 hours ago
@chiragkumarp This PR by itself doesn't really do anything -- can you combine it with #905 and present it as a single PR. Also see the comments on that other PR.